notebooks/post-startup.sh

```diff
@@ -38,12 +38,12 @@ sudo -u "${JUPYTER_USER}" sh -c "/opt/conda/bin/nbstripout --install --global"
 
 # Install & configure the Terra CLI
 sudo -u "${JUPYTER_USER}" sh -c "curl -L https://github.com/DataBiosphere/terra-cli/releases/latest/download/download-install.sh | bash"
+sudo cp terra /usr/bin/terra
 # Set browser manual login since that's the only login supported from an AI Notebook VM.
-sudo -u "${JUPYTER_USER}" sh -c "./terra config set browser manual"
+sudo -u "${JUPYTER_USER}" sh -c "./terra config set browser MANUAL"
```

Comment:
Can you just call `terra` here and below instead of `./terra`?

Reply:
done